### ISSUE TITLE
Infinite shuffle

### ISSUE THREAD

**Issue by TDepinoy** [Meetic/Shuffle#14]
It seems like it is not possible to implement a infinite shuffle.

Any workaround to do that ?

**Comment by tonyshkurenko** [comment]
In my current project, I use this:

    @Override
    public void onViewExited(DraggableView draggableView, Direction direction) {
        final int position = mShuffle.getCurrentAdapterPosition();
        Log.d(TAG, "onViewExited() called with: " + "adapter position = [" + position + "]");

        if (position == mAdapter.getItemCount()) {
             mShuffle.restartShuffling();
        }
        // ... other code
     }

Hope I helped

**Comment by TDepinoy** [comment]
Yes that's the solution I implemented, but I'm facing an issue when my amount of cards is less than the default number of displayed cards (3). In this case, the shuffle is not able to restart correctly.

**Comment by tonyshkurenko** [comment]
Huh, seems, like you've just found a bug in my project 👍  didn't see that, thanks!

**Comment by florent37** [comment]
I'll implement it ASAP, thanks :)

2016-06-14 11:14 GMT+02:00 Anton Shkurenko <notifications@github.com>:

**Comment by florent37** [comment]
I implemented it in the 1.0.8

``` xml
<com.meetic.shuffle.Shuffle
     android:id="@+id/shuffle"
     android:layout_width="match_parent"
     android:layout_height="200dp"
     app:shuffle_infinite="true"
     />
```
